src/loginPage/loginPage.component.tsx

```diff
@@ -88,9 +85,10 @@ interface LoginPageProps {
 interface LoginPageDispatchProps {
   verifyUsernameAndPassword: (
     username: string,
-    password: string
+    password: string,
+    mnemonic: string | undefined,
+    authUrl: string | undefined
```

Comment:
Small style comment, but I prefer the syntax `mnemonic?: string` rather than explicitly saying `mnemonic: string | undefined`